index.html

```diff
@@ -1642,6 +1642,18 @@ <h3>Selective Disclosure</h3>
 all of the credential's claims to be revealed.
 
           </li>
+          <li>
+<b> Hashed Values</b> - With this method, the <a>issuer</a> issue a single
```

Comment:
`<b> Hashed Values</b> - With this method, the <a>issuer</a> issue a single`
-> `<b>Hashed Values</b> &mdash; With this method, the <a>issuer</a> issues a single`

Reply:
Yes fine

Reply:
`<b> Hashed`
-> `<b>Hashed`
was overlooked